bluetooth-rename/index.html

```diff
@@ -161,7 +153,7 @@
               throw "No Web Bluetooth support.";
 
             return navigator.bluetooth.requestDevice({
-              filters: anyDeviceFilter(),
+              acceptAllDevices:true,
```

Comment:
Do you mind adding a comment like https://github.com/GoogleChrome/samples/pull/458/files#diff-ef7cbca5c52cb429b47a308074bf06bcR7?

```diff
+     // filters: [...] <- Prefer filters to save energy & show relevant devices.
```